gax-java/gax-grpc/src/main/java/com/google/api/gax/grpc/GrpcTransportChannel.java

```diff
@@ -60,6 +60,8 @@ public GrpcCallContext getEmptyCallContext() {
   /** The channel in use. */
   abstract ManagedChannel getManagedChannel();
 
+  public abstract boolean isDirectPath();
```

Comment:
```suggestion
  abstract boolean isDirectPath();
```

Reply:
iiuc, this getter method needs to be accessible outside of the package by Spanner. It needs to be public.